apps/examples/wc-vanilla/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "wc-vanilla",
+  "private": true,
+  "version": "0.0.0",
```

Comment:
If you could delete this line it'd be appreciated! This will prevent it from being picked up by changesets